test/testParachainRegistry.t.sol

```diff
@@ -62,23 +71,28 @@ contract ParachainRegistryTest is Test {
         // test non owner trying to register
         vm.prank(nonParaOwner);
         vm.expectRevert("Not owner");
-        registry.register(fakeParaId2, fakePalletInstance2);
+        registry.register(fakeParaId2, fakePalletInstance2, fakeWeightToFee, fakeFeeLocation);
 
         // successful register
         vm.prank(paraOwner2);
-        registry.register(fakeParaId2, fakePalletInstance2);
+        registry.register(fakeParaId2, fakePalletInstance2, fakeWeightToFee, fakeFeeLocation);
 
         // check storage
         ParachainRegistry.Parachain memory parachain = registry.getByAddress(paraOwner2);
         assertEq(parachain.id, fakeParaId2);
         assertEq(parachain.owner, paraOwner2);
         assertEq(parachain.palletInstance, abi.encodePacked(fakePalletInstance2));
+        assertEq(parachain.weightToFee, fakeWeightToFee);
+        assertEq(parachain.feeLocation.parents, fakeFeeLocation.parents);
```

Comment:
Are we not able to assertEq on the whole feeLocation struct rather than individual components?

Reply:
Yes, we are getting error while asserting on Multilocation